commit c7af5452c268ca85c6ee82b7b2ec2761732fbf9f (refs/changes/90/5190/3)
Author: John Levon <john.levon@joyent.com>
Date:   2018-12-11T09:29:07+00:00 (10 months ago)
    
    OS-7431 bump ramdisk size for GCC7
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Approved by: Jerry Jelinek <jerry.jelinek@joyent.com>

diff --git a/tools/build_live b/tools/build_live
index 280c2dd8..022d134c 100755
--- a/tools/build_live
+++ b/tools/build_live
@@ -893,7 +893,7 @@ bi_setup_work_dir
 # Create and mount the "/" (root) ramdisk image:
 #
 bi_file_root="$bi_tmpdir/root.image"
-bi_root_size=278000
+bi_root_size=300000
 if strings "$bi_wsroot/proto/kernel/amd64/genunix" |
     grep "DEBUG enabled" >/dev/null; then
 	#
